### DISCUSSION TITLE
AI疫苗開發平台

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#297] [General]
最近疫情又爆發了，今天公布的最新確診者足跡中新陶芳餐廳就在中央大門口。
雖然變異株Omicorn接連爆出突破性感染，但政府仍希望第三劑疫苗在台灣的覆蓋率能盡快變高，面對疫苗供不應求的情形，宏碁已建構了具高度整合性的AI疫苗開發平台，希望能用AI加速開發，解決疫苗荒。
[施振榮：台灣應扮演「全球疫苗代工中心」](https://www.bnext.com.tw/article/65851/stan-shih)

**Comment by HSU-YU-JEN** [comment]
台灣不管在醫療方面或是疫苗方面的確都極具潛力，希望未來能夠實現成為全球疫苗代工中心CDMO Center

**Comment by MAKERRRofKEVIN** [comment]
之前看過嬌生製藥和Google合作的公司也有在進行這方面的研究與應用，期待台灣也能夠在這方面取得一席之地

**Comment by 107602505** [comment]
台灣在高科技領域舉足輕重，或許AI疫苗開發平台的建立也可以與國外的疫苗合作成為全球性的疫苗代工中心，幫助解決世界各地疫苗供不應求的問題。

**Comment by EricAgumonJou** [comment]
美國國家級的實驗室Inovio約3小時就設計好新疫苗，大幅縮短研發時間也是一個例子。
從基因下手研發疫苗的精準醫療加上AI演算法，相信台灣也有這樣的實力～
期待外來我們台灣研發出來的新型疫苗能幫助更多的人～
